.github/workflows/safety.yml

```diff
@@ -0,0 +1,20 @@
+name: Functional
```

Comment:
I would suggest the workflow is rename to safety to reflect what it actually tests.